### ISSUE TITLE
Latest nightly does not build 0.7.0

### ISSUE THREAD

**Issue by waynenilsen** [tokio-rs/prost#526]
Just to raise awareness. Patch if you want to

rustc version `rustc 1.56.0-nightly (50171c310 2021-09-01)`

```
error[E0034]: multiple applicable items in scope
   --> prost-derive/src/lib.rs:109:14
    |
109 |             .intersperse(quote!(|));
    |              ^^^^^^^^^^^ multiple `intersperse` found
    |
    = note: candidate #1 is defined in an impl of the trait `Iterator` for the type `Map<I, F>`
    = note: candidate #2 is defined in an impl of the trait `Itertools` for the type `T`
help: disambiguate the associated function for candidate #1
    |
105 ~         let tags = Iterator::intersperse(field
106 +             .tags()
107 +             .into_iter()
108 +             .map(|tag| quote!(#tag)), {
109 +         let mut _s = $crate::__private::TokenStream::new();
110 +         $crate::quote_each_token!(_s $($tt)*);
  ...
help: disambiguate the associated function for candidate #2
    |
105 ~         let tags = Itertools::intersperse(field
106 +             .tags()
107 +             .into_iter()
108 +             .map(|tag| quote!(#tag)), {
109 +         let mut _s = $crate::__private::TokenStream::new();
110 +         $crate::quote_each_token!(_s $($tt)*);
  ...

warning: unused import: `itertools::Itertools`
 --> prost-derive/src/lib.rs:9:5
  |
9 | use itertools::Itertools;
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0034`.
warning: `prost-derive` (lib) generated 1 warning
error: could not compile `prost-derive` due to previous error; 1 warning emitted
```

**Comment by stefson** [comment]
rust-1.56.0-beta2 errors with the same error, as I noted when test compiling firefox with it.

**Comment by stefson** [comment]
was this fixed here? https://github.com/tokio-rs/prost/commit/edb1464b71528286a6ce74278af05f19c3e0e820

**Comment by LucioFranco** [comment]
Right yes that commit should have fixed it so upgrading to `0.8` should support the latest nightly. I know some people are having trouble upgrading due to some bugs introduced in 0.8 so I plan on getting 0.9 out pretty soon here.

**Comment by LucioFranco** [comment]
Because this fix is released I am going to close this issue. If there are problems going forward with 0.8/0.9 related to this feel free to reopen.
